.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "node"
+  - "11.10.1" # Due to a bug with 11.11  and TravisCI 
```

Comment:
sorry i took so long to respond to this. looks like this issue is fixed now so we probably don't need to hard code the node version anymore